--- global/certbot-dns-plugins.js
+++ global/certbot-dns-plugins.js
@@ -66,6 +66,16 @@
 		full_plugin_name: 'dns-azure',
 	},
 	//####################################################//
+       bunny: {
+               display_name:        'bunny.net',
+               package_name:        'certbot-dns-bunny',
+               version_requirement: '~=0.0.9',
+               dependencies:        '',
+               credentials:         `# Bunny API token used by Certbot
+dns_bunny_api_key = a65e8ebd-45ab-44d2-a542-40d4d009e3bf`,
+               full_plugin_name: 'dns-bunny',
+       },
+       //####################################################//
 	cloudflare: {
 		display_name:        'Cloudflare',
 		package_name:        'certbot-dns-cloudflare',
